packages/swirl-components/src/components/flip-button/flip-button.mdx

```diff
@@ -1,3 +1,8 @@
+---
+title: Flip-Button
```

Comment:
So far we only used "FlipButton" or "Flip Button" spellings.

Reply:
that was not intentional. revertet it back to FlipButton